pythoncalculator/multiply.py

```diff
@@ -0,0 +1,2 @@
+def multiply(x, y):
+    return x + y
```

Comment:
I think you night have meant '*' rather than '+' .... 

```suggestion
    return x * y
```